src/dex-ui/pages/Pools/Pools.tsx

```diff
@@ -141,60 +200,113 @@ const Pools = (): JSX.Element => {
       : [];
   };
 
+  const getHashScanLink = useCallback(() => {
+    const transactionId = pools.withdrawState.successPayload?.transactionResponse.transactionId.toString();
+    const urlFormattedTimestamp = transactionId?.split("@")[1].replace(".", "-");
+    const formattedTransactionId = `${transactionId?.split("@")[0]}-${urlFormattedTimestamp}`;
+    // format: https://hashscan.io/#/testnet/transaction/0.0.34744739-1665448985-817445871
+    // TODO: set testnet/mainnet based on network
+    return `https://hashscan.io/#/testnet/transaction/${formattedTransactionId}`;
+  }, [pools.withdrawState.successPayload]);
+
   return (
-    <Tabs display={"flex"} flexDirection={"column"}>
-      <TabList display={"flex"} justifyContent={"space-between"} alignItems={"center"} padding={"0"} margin={"0 16px"}>
-        <Flex>
-          {!isEmpty(pools.allPoolsMetrics) ? (
-            <Tab fontSize={"32px"} padding={"0 0 8px 0"} marginRight={"32px"}>
-              Pools
-            </Tab>
-          ) : (
-            ""
-          )}
-          {pools.userPoolsMetrics ? (
-            <Tab fontSize={"32px"} padding={"0 0 8px 0"}>
-              My Pools
-            </Tab>
-          ) : (
-            ""
-          )}
-        </Flex>
-        <Button width={"200px"} marginBottom={"10px"}>
-          Create Pool
-        </Button>
-      </TabList>
-      <TabPanels>
-        <TabPanel>
-          <DataTable colHeaders={colHeadersState.allPoolsColHeaders} rowData={getAllPoolsRowData()} />
-        </TabPanel>
-        <TabPanel>
-          {unclaimedFeeTotal() > 0 ? (
-            <Container
-              padding={"12px"}
-              margin={"0 0 16px 0"}
-              width={"100%"}
-              maxWidth={"100%"}
-              backgroundColor={"#D9D9D9"}
-            >
-              <Flex width={"100%"} justifyContent={"space-between"} alignItems={"center"} flexWrap={"wrap"}>
-                <Text>
-                  Total unclaimed fees across {pools.userPoolsMetrics?.length} pools:&nbsp;
-                  <span style={{ fontWeight: "bold" }}>${unclaimedFeeTotal()}</span>
-                </Text>
-
-                <Button size={"sm"} height={"32px"}>
-                  Claim Fees
-                </Button>
-              </Flex>
-            </Container>
-          ) : (
-            ""
-          )}
-          <DataTable colHeaders={colHeadersState.userPoolsColHeaders} rowData={getUserPoolsRowData()} />
-        </TabPanel>
-      </TabPanels>
-    </Tabs>
+    <>
+      <Tabs
+        display={"flex"}
+        flexDirection={"column"}
+        index={queryParamState.selectedTab}
+        onChange={(index) => setQueryParamState({ ...queryParamState, selectedTab: index })}
+      >
+        {pools.withdrawState.status === "success" && queryParamState.showSuccessfulWithdrawalMessage ? (
```

Comment:
Do we have an existing independent UI component for showing these messages? If not, we should move this into the dex-ui-component library in the future.